icu4c/source/test/intltest/usettest.cpp

```diff
@@ -4722,9 +4722,15 @@ void UnicodeSetTest::TestToPatternOutput() {
             // Escapes are removed even when the syntax is preserved.
             {uR"([ { \x5A e i c h e n k e t t e } [] \x5Aeichenmenge ])",
             u"[{Zeichenkette}[]Zeichenmenge]"},
-            // A named-element is currently a nested set, so it is preserved and causes the syntax to be
-            // preserved.
-            {uR"([ \N{LATIN CAPITAL LETTER Z}eichenmenge ])", uR"([\N{LATIN CAPITAL LETTER Z}eichenmenge])"},
+            // In ICU 78 and earlier, a named-element was a nested set, so it was preserved and
+            // caused the syntax to be preserved.  Now it is treated like an escape.
+            {uR"([ \N{LATIN CAPITAL LETTER Z}eichenmenge ])", uR"([Zceg-imn])"},
+            // This was ill-formed in ICU 78 and earlier (in a convoluted way:
+            // {\N{LATIN CAPITAL LETTER Z} was a well-formed string literal, but then the second }
```

Comment:
Do you mean
```suggestion
            // {\N{LATIN CAPITAL LETTER Z}} was a well-formed string literal, but then the second }
```
?

Reply:
Ok, but there is no “second }” to be unpaired, so I don't understand the second part of the sentence.

Reply:
There is a second `}` in <code>\N{LATIN CAPITAL LETTER Z}eichenkette<b>}</b></code>.

Reply:
I get it now -- there is no second } in the comment, but there is in the test case... nvm